### PR TITLE
Fix CA2229, CA1304

### PR DESCRIPTION
**Pull request by gauravgupta98**
Added constructor in SelectorExpection and fixed CA2229
Added CurrentCulture in JSHandle in .toLower function and fixed CA1304

### CODE REVIEW

File: lib/.editorconfig [hunk 1]
```diff
@@ -149,6 +149,10 @@ csharp_space_between_square_brackets = false
 dotnet_code_quality.ca1802.api_surface = private, internal
 
 # C++ Files
+
+# CS1572: XML comment has a param tag, but there is no parameter by that name
+dotnet_diagnostic.CS1572.severity = none
```
Comment (kblok) [inline lib/.editorconfig:154]: Why do we need this?

Comment (gauravgupta98) [reply lib/.editorconfig:154]: VS suggested this. I thought of suppressing the warning but didn't do. Should I remove this?

**Comment by gauravgupta98** [comment]
Fixed review comments. Please have a look

File: lib/.editorconfig [hunk 2]
```diff
@@ -149,6 +149,7 @@ csharp_space_between_square_brackets = false
 dotnet_code_quality.ca1802.api_surface = private, internal
 
 # C++ Files
+
```
Comment (kblok) [inline lib/.editorconfig:152]:
```suggestion
```

**Comment by gauravgupta98** [comment]
Are we good to merge the changes?

**Comment by kblok** [comment]
Thank you @gauravgupta98!
When I get "Core + Net Framework" green I'll merge it :)

**Comment by gauravgupta98** [comment]
Do I need to change something to get that build to run successfully? 
Sorry, I'm just starting with open source contributions so bit new to the process

**Comment by kblok** [comment]
No worries @gauravgupta98  :)
It takes a while running all tests. The last run was green. So it should get green.

**Comment by kblok** [comment]
Thank you @gauravgupta98 !

**Pull request merged**